CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+1.14.1 / 2019-08-20
```

Comment:
If the latest version in the `CHANGELOG` hasn't been released (normally this can be identified by it not having a release date), the changes can be added under that version. In this case that would be the `1.14.0` version changes.
The detail of the changes only need to be relatively high level and ideally the end state. In this case you can get rid of `Added acceptance folder under test folder` and change `Added feature file with first draft of scenarios for UI tests` to something like `Added UI tests`. The details are able to be retrieved by looking at the details of the commits.

Reply:
Ok that makes sense, we'll make that change.